_includes/medium-loop.html

```diff
@@ -0,0 +1,12 @@
+<div class="row start-xs" id="post-container">
+  <script type="text/template" id="post-template">
+    {% include tile.html tile=site.data.medium tile_type="post" tile_external = true %}
+  </script>
+  <script>
+  $(document).ready(function(){
+    // three posts per row
+    var medium_rows = {{ include.rows | times: 3 }}
+    parseMediumJSON(medium_rows);
```

Comment:
What does times imply here? What does include.rows imply here?

It'd be nice to update parseMediumJSON to use parameters that are easy to read, so instead of passing in the medium_rows object, we could call it with what we might change.

Reply:
I wanted to specify how many rows of content to show (in all of the partials that spit out items) and make sure to fill the row completely. For rows that are 3 columns wide, that means a multiple of 3... so the user would specify 1 row, and that means 3 items, or 2 rows, for 6 items. 

Can you advise me what might make this clearer? It's seemingly not just a matter of renaming the variables, from your comment.